serverless.yml

```diff
@@ -20,6 +20,14 @@ custom:
 stepFunctions:
   stateMachines:
     aqtsCaptureStateMachine:
+      alarms:
+        topics:
+          alarm:
+            Ref: snsTopic
+        metrics:
```

Comment:
These are some of the opinionated alerts provided by the `serverless-step-functions` plugin.